guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md

```diff
@@ -240,6 +264,30 @@ Execute the `LoginCustomerTest.php` and `StorefrontCreateCustomerTest.php` tests
 vendor/bin/mftf run:group -k -- LoginCustomerTest StorefrontCreateCustomerTest
 ```
 
+### `upgrade:tests`
+
+Applies all MFTF major version upgrade scripts to test materials in given path (`test.xml`, `data.xml`, etc).
+
+#### Usage
+
+`upgrade:tests <path>`
+
+`<path>` is the path that contains MFTF test materials that need to be upgraded. The command searches subdirectories for any `*.xml` to attempt to upgrade.
```

Comment:
@KevinBKozan 
Is there a default path?
What if I run `upgrade:tests` with no path?

Reply:
There is no default path, path is required.